lib/nrf_modem_lib/nrf91_sockets.c

```diff
@@ -143,9 +143,6 @@ static int z_to_nrf_level(int z_in_level, int *nrf_out_level)
 	case SOL_SOCKET:
 		*nrf_out_level = NRF_SOL_SOCKET;
 		break;
-	case SOL_DFU:
```

Comment:
I think that this change should also be mentioned in the commit message (removing suppot for deprecated socket types). I might be missing something, but is this anyhow related to the at_cmd/notif?

Reply:
Agree, I split this out in a separate commit